### DISCUSSION TITLE
Фильтры зон

### DISCUSSION THREAD

**Discussion by Urvdmih** [fedorovvl/tso_client#69] [Ideas]
Предлагаю сделать возможность выбрать фильтр по умолчанию или как то закреплять фильтр за приклом.
Мне например снежные приклы выжигают глаза, я всегда ставлю фильтр "none". Но не удобно что после обновления зоны или входа повторного на нее приходится снова этот фильтр выбирать.

**Comment by fedorovvl** [comment]
реализовано в 1.5.3
